tests/phpunit/tests/image/editor.php

```diff
@@ -627,6 +627,47 @@ public function it_should_remove_the_webp_version_of_the_image_if_the_image_is_f
 		);
 	}
 
+	/**
+	 * Test removing the backup sources files when image is removed after edited.
+	 *
+	 * @ticket 55443
+	 */
+	public function it_should_remove_the_backup_sizes_and_sources_if_the_attachment_is_deleted_after_edit() {
```

Comment:
I don't think this test is working. Every test method must either have a name that starts with `test` or an `@test` annotation.

Reply:
nice catch @mukeshpanchal27 ! it indeed wasn't running. I have updated the changes to reflect it in tests now. We need to make sure all other tests are running properly.